src/main/java/org/osc/sdk/controller/api/SdnRedirectionApi.java

```diff
@@ -226,10 +227,11 @@ void updateInspectionHook(InspectionHookElement existingInspectionHook)
      * </p>
      *
      * @param inspectionPort  provides the inspection port to be created
+     * @return an element
```

Comment:
Mentioned that in case of Nsc and Nuage the return element will be null and in case of SFC will have port-group-id as parentId in NetworkElement interface returned.